itl/command-plugins.conf

```diff
@@ -1491,9 +1491,12 @@ object CheckCommand "disk" {
 		"mtmfs",
 		"tracefs",
 		"cgroup",
+		"fuse.*",
+		"fuse",
```

Comment:
As far as I know, `fuse` itself is not a file system that can be used directly, it is more of a file system driver. So why should it be included in the list?